### PR TITLE
[Workers KV] Create advanced-kv-guide.md

### PR DESCRIPTION
**Pull request by vlovich**
Write up tips and tricks that might be relevant to advanced users.

### CODE REVIEW

**Comment by elithrar** [comment]
Kicked off a build so we can get a preview link.

File: content/workers/learning/advanced-kv-guide.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+## Optimizing .get long tail performance
+
+### Embrace long cacheTtl
+
+To optimize the long-tail performance of infrequently accessed keys, specify a longer cacheTtl value (e.g. 1 day).
+Historically, a blocker for many customers was that this meant that your reads wouldn't see writes for the duration of the cacheTtl.
+However, as described in our [architecture blog post](https://blog.cloudflare.com/faster-workers-kv-architecture/), most customers today
+are using the new architecture where you will see updated values within a minute of the write, regardless of the cacheTtl value.
+
+{{<Aside type="note" header="Security considerations"> }}
```
Comment (elithrar) [inline content/workers/learning/advanced-kv-guide.md]:
```suggestion
{{<Aside type="note" header="Security considerations">}}
```

File: content/workers/learning/advanced-kv-guide.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Some customers of Workers KV store authorization tokens. Often time such applications rely on having a strict guarantee on revocation.
+For example, if your service SLA is that a revoked token must be globally revoked within 5 minutes of revocation, your cacheTtl should
+not be longer than 5 minutes. While the write will be noticed within a minute, writes are only noticed due to misses or reads triggering
+a background refresh. If your key is accessed once every 4 minutes and you set a cacheTTL of 10 minutes, it's possible that you will
+exceed your SLA by a few minutes.
+{{</Aside>}}
+
+{{<Aside type="note" header="Availability of long cache TTL noticing writes quickly"> }}
```
Comment (elithrar) [inline content/workers/learning/advanced-kv-guide.md]:
```suggestion
{{<Aside type="note" header="Availability of long cache TTL noticing writes quickly">}}
```

Template tags are finicky.

**Comment by elithrar** [comment]
@vlovich committed a couple of template tag related fixes to unbreak the build.

**Comment by vlovich** [comment]
Maybe the preview link isn't generated for drafts?

**Comment by pedrosousa** [comment]
> Maybe the preview link isn't generated for drafts?

I believe it's not generated for branches in forks.

**Comment by Maddy-Cloudflare** [comment]
Will complete the review on Monday

**Comment by vlovich** [comment]
Superseded by https://github.com/cloudflare/cloudflare-docs/pull/9874 so we can see the preview.

**Pull request closed without merging**